components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -450,7 +451,7 @@ class GeckoEngineSession(
      */
     @Suppress("ComplexMethod")
     private fun createNavigationDelegate() = object : GeckoSession.NavigationDelegate {
-        override fun onLocationChange(session: GeckoSession, url: String?) {
+        override fun onLocationChange(session: GeckoSession, url: String?, geckoPermissions: List<ContentPermission>) {
```

Comment:
I remember we talked about this before. Shouldn't we do something with `geckoPermissions` here. It seems like a bug to ignore that parameter. At a minimum, let's add a comment, please :).

Reply:
Every time that we navigate to site they are passing us the gv permissions for the site, I'm not sure how we could integrate it. As we are querying the store when it's needed and I don't know how useful it's this info at that spesific moment. I'll add a comment for it.